tests/components/bluetooth/test_usage.py

```diff
@@ -57,47 +52,3 @@ async def test_wrapping_bleak_client(
     instance = bleak.BleakClient(MOCK_BLE_DEVICE)
 
     assert not isinstance(instance, HaBleakClientWrapper)
-
-
-async def test_bleak_client_reports_with_address(
```

Comment:
These two tests no longer make sense since the lib takes care of converting it to a BLEDevice if they pass an address so they no longer have to worry about this